js/jquery.mapael.js

```diff
@@ -358,7 +358,15 @@
                 bbox = elem.mapElem.getBBox();
                 textPosition = self.getTextPosition(bbox, elemOptions.text.position, elemOptions.text.margin);
                 elemOptions.text.attrs["text-anchor"] = textPosition.textAnchor;
-                elem.textElem = self.paper.text(textPosition.x, textPosition.y, elemOptions.text.content).attr(elemOptions.text.attrs);
+
+                // Allow fine tuning of the label's position relative to it's calculated position.
+                var dx = 0, dy = 0;
+                if(typeof options.text.attrs.dx !== undefined) dx = options.text.attrs.dx;
```

Comment:
It should be
```javascript
if(options.text.attrs.dx !== undefined)
```
Note: if you are using `typeof`, you should compare to `'undefined'` (notice the quote! `typeof` returns a string).
For Mapael, you can directly use `undefined`.